jwst/fits_generator/templates/nirspec_configuration.gen.inc

```diff
@@ -4,7 +4,7 @@ FILTER    = input('FWA_POS') or 'CLEAR'   /  name of the filter element used
 GRATING   = input('GWA_POS') or 'N/A'  /     name of the grating element used
 GWA_XTIL  = input('GWA_XTIL') or 0.0  /       grating x tilt
 GWA_YTIL  = input('GWA_YTIL') or 0.0  /       grating y tilt
-GWA_TILT  = input('GWA_TTIL') or 0.0 /    GWA temperature ave [K]
+GWA_TILT  = float(input('GWA_TTIL')) or 0.0 /    GWA temperature ave [K]
```

Comment:
Yeah, the NIRSpec ground test images from FITSWriter just love to encode all their numerical values into a string. If I remember correctly they do it with all of the GWA* keyword values, so it might not hurt to make them all robust.